### ISSUE TITLE
508-defect-2 [COGNITION]: Find Form Detail - Spanish content must include a lang="ES" attribute for proper pronunciation with assistive technology

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18031]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh, Trevor_

## User Story or Problem Statement

As a screen reader user who prefers to hear form instructions in Spanish, I want my assistive technology to read instructions with the right pronunciation and cadence.

## Details

Spanish content is not read accurately when the language attribute is defined for English, as we do on all our pages' HTML element. We should add `lang="es"` attributes to the P tags that wrap Spanish language instructions on our find form detail views.

## Acceptance Criteria

- [ ] All Spanish content is identified with a `lang="es"` attribute
- [ ] Zero axe violations with this updated markup
- [ ] Screen readers change pronunciation and shift into Spanish when they encounter properly tagged text. This is usually cued to users by a short pause, and a different tone and inflection of the screen reader voice.

## Environment

* https://staging.va.gov/find-forms/about-form-10-10ez/ (This is likely a template item)

## Proposed Solution (if known)

```diff
<p
  class="vads-u-margin-top--0 vads-u-font-size--h4 vads-u-font-weight--bold"
+ lang="es"
>
  <dfn class="vads-u-visibility--screen-reader">Form name:</dfn>
  Instrucciones y Solicitud de Beneficios Medicos
</p>

- <p>
+ <p lang="es">
  Para que los Veteranos soliciten su afiliación al sistema de atención médica de VA. 
  la información proporcionada en este formulario será utilizada por VA para determinar 
  si usted reúne los requisitos para recibir beneficios médicos.
</p>
```

## WCAG or Vendor Guidance (optional)

* [Language of Parts: Understanding SC 3.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-other-lang-id.html)
* [VSP documentation on accessibility for localization](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/localization/accessibility-for-localization.md)

## Screenshots or Trace Logs

![Screen Shot 2020-12-31 at 3 33 53 PM](https://user-images.githubusercontent.com/934879/103426456-54c41b00-4b7f-11eb-8834-d30ca9e48d7e.png)

**Comment by MarciMcGuireGCIO** [comment]
Closing this as it is a duplicate.
